mon-pix/app/styles/pages/_update-expired-password-form.scss

```
@@ -130,27 +123,31 @@
     display: flex;
     flex-direction: column;
     justify-content: center;
-    align-items: center;
+    align-items: stretch;
     margin: 5px auto 20px;
 
     &--big {
       margin: 10px auto;
-      width: 300px;
+      padding: 0 10px;
+      width: 100%;
 
       @include device-is('desktop') {
         margin: 30px auto;
       }
 
-      @include device-is('mobile') {
-        width: 280px;
-      }
     }
 
     &--margin-top {
       margin-top: 25px;
     }
+    &__button-round{
```

Ici il s'agit d'un modifier dans la nomenclature BEM : 
```suggestion
    &--round{
```

---

Corrigé